frontend/src/components/Login.js

```diff
@@ -1,17 +1,46 @@
 import React, { Component } from 'react';
+
 import '../css/Login.css';
 import '../css/Signup.css';
+import { Route, Redirect, withRouter } from 'react-router-dom';
+import {isAuthenticated} from './LoggedUser';
 
 class Login extends Component {
 
     constructor(props) {
         super(props);
         this.state = { 
             isLoggedIn : false,
+            RedirectLoggedUser: false,
             userType: this.props.match.params.user ? this.props.match.params.user : ""
         };
     }
 
+    // // this will check if user signed out or not
+    // hydrateStatewithLocalStorage() {
+    //   // checks if current value of isLoggedIn is in localStorage and it is true
+    //   if (localStorage.hasOwnProperty("isLoggedIn") && localStorage.getItem("isLoggedIn") === "true")
+    //     isAuthenticated.login();
+    // }
+
+    //calls authenticate which allows user to sign in and view private page
+    login = () => {
+      isAuthenticated.login(() =>
+        this.setState({ RedirectLoggedUser: true })); // set redirect from login page to private to true
+        this.storeLoginUser();
+        console.log(localStorage.getItem("time"));
+        //console.log()
+        //console.log()
+    }
+
+    storeLoginUser = () => {
+        const date = new Date();
+        localStorage.setItem("userEmail", document.getElementById("email"));
+        localStorage.setItem("userType", this.userType)
+        localStorage.setItem("site", "login");
+        localStorage.setItem("time", date.getHours());
+    }
+
```

Comment:
By doing date.getHours(), you'll have a value between 0 and 24 stored in "time". I don't think you'll be able to have a time difference greater than 24 hours if you store time this way.